dist/Dockerfile.e2e/Dockefile

```diff
@@ -0,0 +1,15 @@
+FROM centos:7
+
+ENV HOME="/root"
+
+# Get oc CLI
+RUN mkdir -p ${HOME}/bin && \
+    curl https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/4.3.0-0.nightly-2019-11-13-233341/openshift-client-linux-4.3.0-0.nightly-2019-11-13-233341.tar.gz 2>/dev/null | tar xzf - -C /tmp/bin/ oc
```

Comment:
Do we really need a nightly `oc`?  I'd rather use 4.2.13's or the [floating latest][1] or something.

[1]: https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/

Reply:
Also, this seems to be unpacking into `/tmp/bin`, not `${HOME}/bin`?

Reply:
Floating latest would be best, right, updated